src/swupd_lib/manifest_parser.c

```diff
@@ -364,7 +376,7 @@ struct manifest *manifest_parse(const char *component, const char *filename, boo
 		}
 
 		bitmask_translation = OPTIMIZED_BITMASKS[(unsigned char)line[2]];
-		if (bitmask_translation > AVX512_3) {
+		if (bitmask_translation > APX_7) {
 			error("Skipping unsupported file optimization level\n");
```

Comment:
given our previous lesson should we limit this message to a once kind of thing?

Reply:
Good point, will change that.